<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>439</width>
    <height>325</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QSplitter" name="splitter">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>50</y>
      <width>311</width>
      <height>181</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
    <widget class="QSlider" name="HorizontalSlider">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QProgressBar" name="ProgressBar">
     <property name="value">
      <number>24</number>
     </property>
    </widget>
    <widget class="QLabel" name="ProgressLabel">
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>439</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>HorizontalSlider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>ProgressBar</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>107</x>
     <y>129</y>
    </hint>
    <hint type="destinationlabel">
     <x>113</x>
     <y>180</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>HorizontalSlider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>ProgressLabel</receiver>
   <slot>setNum(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>218</x>
     <y>142</y>
    </hint>
    <hint type="destinationlabel">
     <x>221</x>
     <y>244</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
